stylesheet.css

```diff
@@ -0,0 +1,29 @@
+body {
+	background-color: DarkSlateBlue ;
+}
+
+.background-image {
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: 1;
+  background-image: url('\zaraki.png');
```

Comment:
Lets use something more neutral as background

Reply:
Changed to more neutral bg, please let me know if it doesn't fit too